src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
```

Comment:
Add some documentation to the class. For example, what is the purpose of this class